haskell-language-server.cabal

```diff
@@ -19,6 +19,19 @@ extra-source-files:
   README.md
   ChangeLog.md
   include/ghc-api-version.h
+  test/testdata/**/*.project
+  test/testdata/**/*.cabal
+  test/testdata/**/*.yaml
+  -- this one is not matched by the previous glob
+  test/testdata/hlint/ignore/.hlint.yaml
+  test/testdata/**/*.h
+  test/testdata/**/*.hs
+  test/testdata/**/*.expected
+  test/testdata/**/*.error
+  -- eval plugin test data is used in the test suite
+  plugins/hls-eval-plugin/test/testdata/*.yaml
+  plugins/hls-eval-plugin/test/testdata/*.hs
+  plugins/hls-eval-plugin/test/testdata/*.hs.expected
```

Comment:
This is kinda hacky to me... Let's move eval plugin's tests into its own package, like Sandy did in #1425

Reply:
Agree but it deserves its own pr? This one only makes the hack more evident :smiley:

Reply:
@berberman we can wait for such pr and rebase, if it is about to be done, of course

Reply:
I'm working on this. I think we need extract `hls-test-utils` to a standalone package, with some test framework stuffs re-exported, so that tests extracted from HLS could reuse these code.

Reply:
I will remove eval test data files and wait for #1604